app/models/foreman_dlm/dlmlock_event.rb

```diff
@@ -0,0 +1,20 @@
+module ForemanDlm
+  class DlmlockEvent < ApplicationRecord
+    include Authorizable
```

Comment:
if we include `Authorizable`, we also have to define permissions for this.

Reply:
Done